### DISCUSSION TITLE
2022-03-23 Weekly Meeting 48 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#201] [Weekly Meetings]
Weekly Meeting 48 is happening on Wednesday March 16th, 2022 from 9:00 AM PDT - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Announcements and Updates
- Recap previous week workshop on OMI structure and leadership.
- https://miro.com/app/board/uXjVOFdPAIQ=/?invite_link_id=103696866511

Working Group Updates:

- OMI Website update
- OMI-Chan group
- Scripting group
- gLTF group
- any others?

 
Please leave any additional agenda items below in the comments. Thanks!

**Comment by mrmetaverse** [comment]
Backlog grooming moved to Monday 3/28/22 for this month at 21:05 UTC

**Comment by mrmetaverse** [comment]
Need someone to confirm the calendar is in fact fixed. I tried to adjust due to daylight savings, but saw chatter that there were conflicting times in folks calendars. 

1. Review this ticket: https://github.com/omigroup/omigroup/issues/202
2. Assign someone to fix it 
3. Move ticket to done when done

**Comment by mrmetaverse** [comment]
I might miss todays meeting due to poor connection on the mountain :mountain: If so, my main update is that we will resume our discussion on the [Future Structure of OMI](https://github.com/omigroup/omigroup/milestone/2) next week. **If you have any action items to add, related to the future structure, [please use the github milestone](https://github.com/omigroup/omigroup/milestone/2) to ensure we are tracking everything in the right place (and not just impromptu discussions).**

**Comment by robertlong** [comment]
The OMI GDC Meetup is at 5PM today!

Hoping to see some of you there!

![image](https://user-images.githubusercontent.com/1753624/159743683-fc96dc74-5261-4d46-8036-c41f99f699e2.png)
